### ISSUE TITLE
scylla cdc library failed upon schema change

### ISSUE THREAD

**Issue by Prateekdecp** [scylladb/scylla-cdc-java#83]
I am trying to use the scylla cdc driver library for deriving all cdc logs for a table
After I added a new column to the existing table,
the driver library gave me an error and crashed
Method threw 'shaded.com.scylladb.cdc.driver3.driver.core.exceptions.InvalidTypeException' exception. Cannot evaluate com.scylladb.cdc.cql.driver3.Driver3RawChange.toString()
Upon restart I started getting the entries as expected.
Is there a fix for this?

**Comment by dkropachev** [comment]
> I am trying to use the scylla cdc driver library for deriving all cdc logs for a table After I added a new column to the existing table, the driver library gave me an error and crashed Method threw 'shaded.com.scylladb.cdc.driver3.driver.core.exceptions.InvalidTypeException' exception. Cannot evaluate com.scylladb.cdc.cql.driver3.Driver3RawChange.toString() Upon restart I started getting the entries as expected. Is there a fix for this?

@Prateekdecp , can you please provide table definition before and after.

**Comment by Prateekdecp** [comment]
create table temp_keyspace.users(id primary key,name text,phone_number text);
Initial table

alter table  temp_keyspace.users add email text;
altered table

**Comment by dkropachev** [comment]
Depends on https://github.com/scylladb/java-driver/issues/460

**Comment by dkropachev** [comment]
@Bouncheck , was this addressed in https://github.com/scylladb/scylla-cdc-java/pull/90 ?

**Comment by Bouncheck** [comment]
Yes, after the driver update the InvalidTypeException does not appear in this scenario.
